### PR TITLE
Add sync_only command

### PR DESCRIPTION
**Pull request by borgius**

### CODE REVIEW

File: src/docker-osx-dev [hunk 1]
```diff
@@ -956,12 +958,19 @@ function configure_includes {
...
 #
 function watch_and_sync {
+  sync_only
+  watch
+}
+
+#
+# Runs the docker-osx-dev script to syncing one time.
+#
+function sync_only {
```
Comment (brikis98) [inline src/docker-osx-dev:968]: Given that we have functions called `watch` and `watch_and_sync`, it seems like `sync` would be a better name here.

**Comment by edenvicary** [comment]
We will benefit from this feature! :)

**Comment by brikis98** [comment]
@edenvicary: agreed! I'm just waiting on one minor change from @borgius before merging.

**Comment by ain** [comment]
:+1:

**Comment by edenvicary** [comment]
Good stuff @brikis98, @borgius we need you! :wink:

**Comment by brikis98** [comment]
Looks like this was handled in bed86264b21a0793565e61723b15805c96ae1101. Closing.

**Pull request closed without merging**